### PR TITLE
New fields to config

### PR DESCRIPTION
**Pull request by marko1777**

### CODE REVIEW

File: .github/workflows/windows-build.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    steps:
+      - name: Setup | Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup | Artifacts
+        uses: actions/download-artifact@v2
+
+      - name: Setup | Checksums
+        run: for file in $(find ./ -name '*.dll' ); do openssl dgst -sha256 -r "$file" | awk '{print $1}' > "${file}.sha256"; done
```
Comment (outspace) [inline .github/workflows/windows-build.yml:57]: change *.dll to *.exe

File: src/uapi/openbsd/net/if_wg.h [hunk 2]
```diff
@@ -67,20 +67,39 @@ struct wg_peer_io {
...
+#define WG_INTERFACE_DEVICE_HAS_H2	 (1 << 11)
+#define WG_INTERFACE_DEVICE_HAS_H3	 (1 << 12)
+#define WG_INTERFACE_DEVICE_HAS_H4	 (1 << 13)
 
 struct wg_interface_io {
-	uint8_t			i_flags;
+	uint16_t		i_flags;
 	in_port_t		i_port;
-	int			i_rtable;
+	int				i_rtable;
```
Comment (outspace) [inline src/uapi/openbsd/net/if_wg.h:88]: format

File: src/uapi/openbsd/net/if_wg.h [hunk 3]
```diff
@@ -40,8 +40,8 @@ struct wg_aip_io {
 #define WG_PEER_HAS_PUBLIC		(1 << 0)
 #define WG_PEER_HAS_PSK			(1 << 1)
 #define WG_PEER_HAS_PKA			(1 << 2)
-#define WG_PEER_HAS_ENDPOINT		(1 << 3)
-#define WG_PEER_REPLACE_AIPS		(1 << 4)
+#define WG_PEER_HAS_ENDPOINT	(1 << 3)
```
Comment (outspace) [inline src/uapi/openbsd/net/if_wg.h:43]: format

Comment (marko1777) [reply src/uapi/openbsd/net/if_wg.h:43]: image
I think I changed it from the left to right.
I don't understand why it's like that

File: src/uapi/openbsd/net/if_wg.h (hunk 2 above)
Comment (marko1777) [reply src/uapi/openbsd/net/if_wg.h:88]: same

Comment (outspace) [reply src/uapi/openbsd/net/if_wg.h:88]: use spases (not tabs)

File: src/uapi/openbsd/net/if_wg.h (hunk 3 above)
Comment (outspace) [reply src/uapi/openbsd/net/if_wg.h:43]: use spaces not tabs

**Pull request merged**